app/models/DepartureSelector.scala

```diff
@@ -49,8 +49,8 @@ object MessageSelector {
     messageSelector =>
       Json.obj(
         "$and" -> Json.arr(
-          Json.obj("_id"                                                 -> messageSelector.departureId),
-          Json.obj(s"messages.${messageSelector.messageId.index}.status" -> Json.obj("$exists" -> true))
+          Json.obj("_id"                                                     -> messageSelector.departureId),
+          Json.obj(s"messages.${messageSelector.messageId.index - 1}.status" -> Json.obj("$exists" -> true))
```

Comment:
This might be something that we move into the MessageId model

Reply:
Yep, I was thinking about changing the property of the case class to `value` and having a separate `def index`, thanks for the reminder 👍